start/routes/auth.ts

```diff
@@ -0,0 +1,6 @@
+import Route from '@ioc:Adonis/Core/Route'
+
+Route.group(() => {
+  Route.post('auth/signin', 'AuthController.signIn')
+  Route.post('auth/signup', 'AuthController.signUp')
+}).prefix('api')
```

Comment:
```ts
Route.group(() => {
  Route.post('signin', 'AuthController.signIn')
  Route.post('signup', 'AuthController.signUp')
}).prefix('auth/api')
```

Reply:
Please make similiar changes in other route files

Reply:
Done